### DISCUSSION TITLE
Experimental build of v5 when?

### DISCUSSION THREAD

**Discussion by insertnameuserhere** [otomad/OtomadHelper#75] [Q&A] [answered]
if that's going to happen make sure your using the "Pre-release" tag

**Comment by otomad** [comment, marked as answer]
Yeah, the new version may be unstable and may cause issues such as Vegas crashes. It is expected that the beta versions will be released with the Pre-release tag in the early stages for everyone to experience and discover bugs. And it will be released once it stabilizes.

**Reply by insertnameuserhere** [reply]
k thx

**Reply by insertnameuserhere** [reply]
> Yeah, the new version may be unstable and may cause issues such as Vegas crashes. It is expected that the beta versions will be released with the Pre-release tag in the early stages for everyone to experience and discover bugs. And it will be released once it stabilizes.

oh also when does it release?

**Reply by otomad** [reply]
> oh also when does it release?

I'm not sure yet, but I'm currently working on coding the new UI and there's still a lot of work to be done.

**Reply by otomad** [reply]
If you want to keep up with the latest progress of the project, you can switch to the new branch:

https://github.com/otomad/OtomadHelper/blob/webview/

**Reply by insertnameuserhere** [reply]
> If you want to keep up with the latest progress of the project, you can switch to the new branch:
> 
> https://github.com/otomad/OtomadHelper/blob/webview/

![image](https://github.com/user-attachments/assets/2eb20c31-d44a-406f-9440-e687b44be6ec)
help

**Reply by otomad** [reply]
> help

Oops, I have not finished it. It just the source code, not the extension itself.
